packages/office-addin-lint/test/src/test.ts

```diff
@@ -13,10 +13,29 @@ describe("test cases", function() {
         const lintExpectedCommand = "./test/cases/basic/functions.ts";
         const lintFixExpectedCommand = "--fix ./test/cases/basic/functions.ts";
         const prettierExpectedCommand = "--parser typescript --write ./test/cases/basic/functions.ts";
+
+        const lintCheckCommand = prettierLint.getLintCheckCommand(inputFile);
+        assert.strictEqual(lintCheckCommand.indexOf(lintExpectedCommand) > 0 , true, "Lint command does not match expected value.");
+
+        const lintFixCommand = prettierLint.getLintFixCommand(inputFile);
+        assert.strictEqual(lintFixCommand.indexOf(lintFixExpectedCommand) > 0, true, "Lint fix command does not match expected value.");
+
+        const prettierCommand = prettierLint.getPrettierCommand(inputFile);
+        assert.strictEqual(prettierCommand.indexOf(prettierExpectedCommand) > 0, true, "Prettier command does not match expected value.");
+    });
+
+    it("spaces in filename", async function() {
+        const inputFile = "./test/cases/basic/functions with space.ts";
```

Comment:
Did we also verify for spaces in file path (e.g. "/test/cases/**basic cases**/functions with space.ts"?